.github/scripts/update_tools.py

```diff
@@ -7,25 +7,20 @@
 """
 
 from pathlib import Path
-import os
-import sys
 import yaml
 import requests
 
 # Add the root directory to the Python path
-root_dir = Path(__file__).resolve().parents[2]
+ROOT_DIR = Path(__file__).resolve().parents[2]
+YAML_PATH = ROOT_DIR / "versions.yaml"
 
 def load_packages():
     """Load package configurations from versions.yaml file."""
-    yaml_path = os.path.join(root_dir, "versions.yaml")
     try:
-        with open(yaml_path, 'r', encoding='utf-8') as file:
-            data = yaml.safe_load(file)
-        # Expect data["packages"] to be a dict
+        data = yaml.safe_load(Path(YAML_PATH).read_text(encoding="utf-8"))
         packages = data.get("packages", {})
-        data["packages"] = packages
-        return data
-    except yaml.YAMLError as error:
+        return data, packages
```

Comment:
To me this one could just return the "data" and then it's the caller that do the `.get`